packages/dashboards/src/lib/components/timeseries-widget/chart-presets/status-bar-chart/status-bar-chart.component.ts

```diff
@@ -318,6 +318,23 @@ export class StatusBarChartComponent
         }
     }
 
+    public displayDeleteButton(): boolean {
+        return (
+            !!this.configuration.allowLegendMenu &&
+            this.configuration.projectType ===
+                TimeseriesWidgetProjectType.PerfstackApp
+        );
+    }
+
+    public removeMetric(metricId: string): void {
```

Comment:
you can move this method to TimeseriesChartComponent and use it from both status and xy.